src/main/webapp/app/pages/teamPage/TeamPage.tsx

```diff
@@ -1,15 +1,19 @@
 import * as React from 'react';
 import { Col, Row } from 'react-bootstrap';
 import DocumentTitle from 'react-document-title';
-import _ from 'lodash';
```

Comment:
I don't think this file needs to be reviewed. Mostly shuffle around the names and add a member type for past contributor,